articles/data-binding-with-angular/index.md

```diff
@@ -0,0 +1,301 @@
+**Angular** is a platform and design framework for building a SPA using HTML and TypeScript.
+ Angular is written in TypeScript(although the first version, i.e.AngularJS, was written in JS).
+ It implements its core and optional functionalities as a set of TypeScript libraries that 
+ you import into your apps.
+
+Before diving into the tutorial, we will have a look at
+ what *Modules* and *Components* are.
+
+### Components
+Components are the most fundamental building block of any Angular app.
```

Comment:
Lets keep breaking this down. 
Perhaps it would be helpful to add a terminology section to go over all these terms (and future terms) that will be needed to know to best follow along.